rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1327,6 +1337,7 @@ def test_upgrade_db_35_to_36(user_data_dir):  # pylint: disable=unused-argument
     res = cursor.execute('SELECT * FROM history_events WHERE identifier=?', (234,)).fetchone()
     assert res is not None
     assert res[9] == 'Edited event'  # event's notes
+    db.logout()
```

Comment:
instead of doing this in every single test does it not make sense to do this as part of the DB fixture? Or wait ... it's actually there: https://github.com/rotki/rotki/blob/ebf85137354eec299e6bc541efa91ccaa1e943ad/rotkehlchen/tests/fixtures/db.py#L168

So this is not needed at the end of each test, as long as it's the DB intiialized by the fixture

Reply:
yeah right, we only need to add it in the tests that create a db as part of the test